pages/management/ManagementHandler.php

```diff
@@ -586,19 +604,17 @@ public function manageEmails(array $args, Request $request): void
 
     /**
      * Edit or view user using user access table action
-     * @param array $args
-     * @param $request
-     * @return void
+     *
      * @throws \Exception
      */
     public function editUser(array $args, $request): void
     {
         $this->setupTemplate($request);
         $userId = $args[0];
-        if(!empty($userId)) {
+        if (!empty($userId)) {
```

Comment:
In general I think it's better to invert the condition (the `handle404()` will quit the application), then we'll have a smaller indentation and be able to drop the `else`.